scripts/cli/reportI18nStats.mjs

```diff
@@ -0,0 +1,87 @@
+/// @ts-check
+
+import { readdir, stat, readFile } from 'fs/promises';
+import path from 'path';
+
+const LOCALES_DIR = path.join('.', 'public', 'locales');
```

Comment:
better if you start building this route from `__dirname` and `path.resolve` so you work with absolute paths

Reply:
nice!